core/src/main/scala/com/yahoo/maha/core/query/postgres/PostgresQueryGenerator.scala

```diff
@@ -1034,21 +1034,21 @@ b. Dim Driven
       val hasPartitioningScheme = fact.annotations.contains(PostgresQueryGenerator.ANY_PARTITIONING_SCHEME)
 
       //add subquery
-      if(isFactOnlyQuery) {
-        queryContext.dims.foreach {
-          subqueryBundle =>
-            val factFKCol = fact.publicDimToForeignKeyMap(subqueryBundle.publicDim.name)
-            val factFkColAlias = {
-              if (fact.columnsByNameMap.contains(factFKCol)) {
-                fact.columnsByNameMap.get(factFKCol).get.alias
-              } else {
-                None
-              }
-            }
-            val sql = generateSubqueryFilter(factFkColAlias.getOrElse(factFKCol), filters, subqueryBundle, queryBuilderContext)
-            whereFilters += sql
-        }
-      }
+//      if(isFactOnlyQuery) {
```

Comment:
Commenting this out because subqueries for dim-fact query type are handled by ogb generator.